src/framework/global/types/string.h

```diff
@@ -428,6 +429,22 @@ class AsciiStringView
 
 inline String operator+(char16_t s1, const String& s2) { String t(s1); t += s2; return t; }
 inline String operator+(const char16_t* s1, const String& s2) { String t(s1); t += s2; return t; }
+
+// ============================
+// StringCodec
+// ============================
+
+class StringCodec
```

Comment:
Do we need this class? We've already had similar functionality in mu::String class

Reply:
Now this class is not needed